### PR TITLE
Standardized transfer code

### PR DESCRIPTION
**Pull request by chenjiandongx**

### CODE REVIEW

**Comment by chenjiandongx** [comment]
@710leo @UlricQin Another PR for transfer module.

File: src/modules/transfer/http/routes/health_router.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func tsdbInstance(c *gin.Context) {
 
 	pk := dataobj.PKWithCounter(input.Endpoint, counter)
 	pools, err := backend.SelectPoolByPK(pk)
-	addrs := []string{}
+	addrs := make([]string, len(pools))
```
Comment (UlricQin) [inline src/modules/transfer/http/routes/health_router.go:58]: 这里应该是：addrs := make([]string, 0, len(pools))

**Comment by chenjiandongx** [comment]
@UlricQin @710leo 

Thanks for your review as I just realize that I have made this mistake in the previous PR(https://github.com/didi/nightingale/commit/0604c690d429deedcf50a3b1e9ed0042291b6738). I'm about to fix it, but I find it had been corrected by @710leo at the commit(https://github.com/didi/nightingale/commit/56f8241ea51cd570621585124d5aa75c58d21256). 

Team works! 👍

File: src/modules/transfer/backend/query.go [hunk 2]
```diff
@@ -189,9 +190,11 @@ func fetchData(start, end int64, consolFun, endpoint, counter string, step int)
 		return resp, err
 	}
 
+	// 没有获取到数据 做标记处理 math.NaN()
 	if len(resp.Values) < 1 {
 		ts := start - start%int64(60)
 		count := (end - start) / 60
+		// TODO(mark): threshold
```
Comment (710leo) [inline src/modules/transfer/backend/query.go:197]: 这里加 `TODO` 的目的是？

Comment (chenjiandongx) [reply src/modules/transfer/backend/query.go:197]: 喔 忘了删了 当时我是想我等下回过头来思考一下这个数值的意义

Comment (chenjiandongx) [reply src/modules/transfer/backend/query.go]: 新 commit 已经移除了

**Pull request merged**